doc/getting_started/index.rst

```diff
@@ -61,116 +50,100 @@ tools. First, make sure your development system OS is updated:
 
-         a) Add the kitware signing key to apt:
+         a) Add the Kitware signing key:
 
             .. code-block:: bash
 
                wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | sudo apt-key add -
 
-         b) Add the kitware repo corresponding to the Ubuntu 18.04 LTS release:
+         b) Add the Kitware apt repository for your OS release. For Ubuntu
+            18.04:
 
             .. code-block:: bash
 
                sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main'
 
-         c) Then install the updated cmake using the usual apt commands:
+         c) Then install the updated cmake with ``apt``:
 
             .. code-block:: bash
 
                sudo apt update
                sudo apt install cmake
 
-      #. Install west:
+      #. Install west, and make sure :file:`~/.local/bin` is on your
+         :envvar:`PATH` :ref:`environment variable <env_vars>`:
 
          .. code-block:: bash
 
             pip3 install --user -U west
             echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
             source ~/.bashrc
 
-         The pip3 ``--user`` option puts installed Python packages into your
-         ``~/.local/bin folder`` so we'll need to add this to the PATH
-         so these packages will be found.  Adding the PATH specification to your
-         ``.bashrc`` file ensures this setting is permanent.
-
    .. group-tab:: macOS
 
-      #. On macOS, install :program:`Homebrew` by following instructions on the `Homebrew
-         site`_, and as shown here. Homebrew is a free and open-source package management system that
-         simplifies installing software on macOS.  While installing Homebrew,
-         you may be prompted to install additional missing dependencies; please follow
-         any such instructions as well.
+      #. Install `Homebrew <https://brew.sh/>`_:
 
          .. code-block:: bash
 
             /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 
-      #. Then, install these host dependencies with the ``brew`` command:
+      #. Use ``brew`` to install dependencies:
 
          .. code-block:: bash
 
-            brew install cmake ninja gperf ccache dfu-util qemu dtc python3
+            brew install cmake ninja gperf python3 ccache
 
       #. Install west:
 
          .. code-block:: bash
 
             pip3 install west
 
-      .. _Homebrew site: https://brew.sh/
-
    .. group-tab:: Windows
 
-      .. note:: Currently, the built-in `Windows Subsystem for Linux (WSL)
+      .. note:: We don't recommend using the `Windows Subsystem for Linux (WSL)
          <https://msdn.microsoft.com/en-us/commandline/wsl/install_guide>`__
-         doesn't support flashing your application to the board.  As such,
-         we don't recommend using WSL yet.
+         for Zephyr development yet, because it's harder to flash Zephyr
+         applications to boards using WSL.
```

Comment:
Not supported, or just harder ?
If it is just harder, should we make a page describing, how to do it ?

Reply:
A user reported it's possible if you manually rename some of the underlying flash utilities. I haven't figured it out or even tried. I think that's future work and the recommendation should stand; I'm just trying to avoid a potentially inaccurate claim that it "can't" be done since users are claiming it can.

Reply:
Ok, thanks for the explanation.
Then howabout something like:

```
The Zephyr project does not support flashing of your application using the
built-in Windows Subsystem for Linux (WSL).

Therefore we don't recommend using the `Windows Subsystem for Linux (WSL).
```
that way we only say we are not supporting it, which is different from: It cannot be done.